frappe/commands/site.py

```diff
@@ -284,7 +285,12 @@ def migrate(context, rebuild_website=False, skip_failing=False):
 		frappe.init(site=site)
 		frappe.connect()
 		try:
-			migrate(context.verbose, rebuild_website=rebuild_website, skip_failing=skip_failing)
+			migrate(
+				context.verbose,
+				rebuild_website=rebuild_website,
+				skip_failing=skip_failing,
+				skip_search_index=skip_search_index
```

Comment:
We should remove building search index in migrate.

Move it to a separate command:
`bench --site sitename build-search-index`